.github/workflows/notify.yml

```
@@ -0,0 +1,10 @@
+name: Draco Insights Dashboard Notify
+on:
+  pull_request:
```

```suggestion
  pull_request: ["Created"]
```

I think this is what you want. Otherwise, it'll run every time someone pushes to the pull request.